docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Setting up the Okta Service as a hosted integration
+description: How to set up the Okta Service as a hosted integration
+---
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- An Okta organization with an [Okta API token](https://developer.okta.com/docs/guides/create-an-api-token) created.
+
+<Details title="Okta API token permissions">
+(!docs/pages/includes/okta-permissions.mdx hosted="true"!)
+</Details>
+
+### Enrolling a hosted Okta Integration
```

Comment:
Could we add a few sentence on what Enrolling a hosted Okta Integration does and what is the business value in  SCIM okta sync ?

Reply:
Yep, was going to ask the same: we need to say what "hosted" means and that this is a recommended way to setup Okta integration. In fact, I would probably mark the other, self-hosted, guide as deprecated as we want to get rid of it in 16. Maybe add a note on the top of the page and redirect here.